app/core/templates/core/challenge_detail.html

```diff
@@ -21,6 +29,27 @@ <h5>Your submissions</h5>
       {% endif %}
     </p>
 
+    {% if elements %}
+    <p class="card-text">
+      <h5>Public Input Data</h5>
```

Comment:
We discussed this and agreed not to allow this, but I wanted to discuss again since it seems like it would be really convenient.